packages/components/psammead-caption/src/index.stories.jsx

```diff
@@ -41,8 +41,9 @@ storiesOf('Components|Caption', module)
       ],
       ([linkText, captionText], script) => (
         <Caption script={script}>
-          {captionText}
-          <InlineLink href="https://www.bbc.com"> {linkText}</InlineLink>.
+          {`${captionText} `}
+          <InlineLink href="https://www.bbc.com">{linkText}</InlineLink>
+          {` ${captionText} `}
```

Comment:
Why is caption text here twice?

Reply:
As per the `Code changes` in the description of the PR:
>Adds a second line of text so the inline link has both text content before and after it